### PR TITLE
(SIMP-2462) Update Versions

### PR DESCRIPTION
**Pull request by jeannegreulich**
updated the requires and metadata.json files

SIMP-2462 #comment simp_grafana updated

### CODE REVIEW

**Comment by jeefberkey** [comment]
Updating this module

**Review by nick-markowski — CHANGES_REQUESTED** [review]
I'm making a patch. It's gonna be a great patch; the best patch.

File: manifests/params.pp [hunk 1]
```diff
@@ -7,24 +7,27 @@
...
 
-  $base_dn = hiera('ldap::base_dn', 'dc=invalid')
-  $bind_dn = hiera('ldap::bind_dn', "uid=%s,${base_dn}")
-  $bind_pw = hiera('ldap::bind_pw', undef)
+  $app_pki_dir             = '/etc/pki/simp_apps/grafana/x509'
+  $app_pki_key             = "${app_pki_dir}/private/${facts['fqdn']}.pem"
+  $app_pki_cert            = "${app_pki_dir}/public/${facts['fqdn']}.pub"
 
-  $ldap_urls   = hiera_array('ldap::uri', [''])
+  $base_dn = simplib::lookup('simp_options::ldap::base_dn', { 'default_value' => 'dc=invalid' } )
```
Comment (nick-markowski) [inline manifests/params.pp]: simplib::ldap::domain_to_dn?

**Review by nick-markowski — CHANGES_REQUESTED** [review]

File: manifests/params.pp (hunk 1 above)
Comment (jeefberkey) [reply manifests/params.pp]: What's that?

Comment (nick-markowski) [reply manifests/params.pp]: The new default for base_dn

**Review by nick-markowski — CHANGES_REQUESTED** [review]
I'll merge after a changelog update

**Review by nick-markowski — APPROVED** [review]

**Pull request merged**